components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ThumbnailLoader.java

```diff
@@ -385,18 +380,21 @@ private ThumbnailStorePrx getThumbnailStore(PixelsData pxd) throws DSAccessExcep
     }
 
     /**
-     * Returns whether a pyramid should be used for the given {@link PixelsData}.
-     * This usually implies that this is a "Big image" and therefore will
-     * need tiling.
+     * Returns whether a pyramid should be used for the given {@link PixelsData}
+     * . This usually implies that this is a "Big image" and therefore will need
+     * tiling.
      *
      * @param pxd
      * @return
      */
-    private boolean requiresPixelsPyramid(PixelsData pxd) throws ServerError, DSOutOfServiceException {
-        int maxWidth = Integer.parseInt(getConfigService()
-                .getConfigValue("omero.pixeldata.max_plane_width"));
-        int maxHeight = Integer.parseInt(getConfigService()
-                .getConfigValue("omero.pixeldata.max_plane_height"));
+    private boolean requiresPixelsPyramid(PixelsData pxd) throws ServerError,
+            DSOutOfServiceException {
+        int maxWidth = context.lookup(LookupNames.MAX_PLANE_WIDTH) != null ? Integer
+                .parseInt((String) context.lookup(LookupNames.MAX_PLANE_WIDTH))
+                : 3192;
+        int maxHeight = context.lookup(LookupNames.MAX_PLANE_HEIGHT) != null ? Integer
+                .parseInt((String) context.lookup(LookupNames.MAX_PLANE_HEIGHT))
+                : 3192;
         return pxd.getSizeX() * pxd.getSizeY() > maxWidth * maxHeight;
```

Comment:
I'm actually not sure. Is this really true? Or should that rather be `return pxd.getSizeX() > maxWidth || pxd.getSizeY() > maxHeight` ?

Reply:
I could do some experiments for you if you like.

Reply:
As @joshmoore pointed out `pxd.getSizeX() * pxd.getSizeY() > maxWidth * maxHeight` is indeed correct.

Reply:
the size of the plane matters 
``pxd.getSizeX() * pxd.getSizeY() > maxWidth * maxHeight`` is correct